src/lang/en.json

```
@@ -29,7 +29,8 @@
     "statusMaintenance": "Maintenance",
     "Maintenance": "Maintenance",
     "Unknown": "Unknown",
-    "Cannot connect to the socket server": "Cannot connect to the socket server",
+    "Lost connection to the socket server. Reconnecting...": "Lost connection to the socket server. Reconnecting...",
+    "Cannot connect to the socket server.": "Cannot connect to the socket server.",
```

See below, this is just a combination of two senences

```suggestion
    "Cannot connect to the socket server": "Cannot connect to the socket server",
```